esp8266/Makefile

```diff
@@ -148,7 +148,7 @@ DRIVERS_SRC_C = $(addprefix drivers/,\
 SRC_S = \
 	gchelper.s \
 
-FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
+FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py') modules/upip.py modules/upip_utarfile.py
```

Comment:
> missed the point where the symlink would be in the git repo

Yes, sorry, there was a lot assumed by context, perhaps I should have made it clear when asked you.

> ..or the windows user in this case.

Right, the maintainer can say that ;-). The point is that I, as a change author, don't want to knowlingly or unknowingly introduce a problem which may hit someone later, so ask for a consult. Thanks for actually checking it - now we're all set. (And in many cases, just ack/nak from whatever you already know is good enough.)